src/test/java/com/synopsys/integration/jenkins/detect/substeps/DetectJarManagerTest.java

```diff
@@ -26,82 +21,115 @@
 
 public class DetectJarManagerTest {
 
-    private final static String testDetectJarPath = "/test/detect/jar/path/detect.jar";
-    private final static String testJavaHome = "/test/java/home/path";
-    private final static String testPathEnv = "/test/path/env";
+    private final static String detectJarPathInput = "/test/detect/jar/path/detect.jar";
+    private final static String javaHomeInput = "/test/java/home/path";
+    private final static String javaBinPath = "/bin/java";
+    private final static String javaPath = javaHomeInput + javaBinPath;
+    private final static String javaPathSystem = System.getProperty("user.dir") + "/ " + javaBinPath;
+    private final static String pathEnvVarInput = "/test/path/env";
+
+    private final static Map<String, String> environmentVariables = new HashMap<>();
+
+    private final TaskListener taskListener = Mockito.mock(TaskListener.class);
+    private final JenkinsIntLogger jenkinsIntLogger = new JenkinsIntLogger(taskListener);
 
     private final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-    private final PrintStream originalOut = System.out;
 
     @BeforeEach
     public void setUp() {
-        System.setOut(new PrintStream(byteArrayOutputStream));
+        environmentVariables.put("PATH", pathEnvVarInput);
+        Mockito.when(taskListener.getLogger()).thenReturn(new PrintStream(byteArrayOutputStream));
```

Comment:
Is there any particular reason that we're doing this work here as opposed to in createAndTestDetectSetupResponse?